.github/workflows/docker_scan.yml

```diff
@@ -29,7 +32,7 @@ jobs:
         supported_image_tag: ['latest', '3.7.0']
     steps:
       - name: Run CVE scan
-        uses: aquasecurity/trivy-action@master
+        uses: aquasecurity/trivy-action@d9cd5b1c23aaf8cb31bb09141028215828364bbb # master
```

Comment:
This seems like a very important change. We should not be pulling in `master` of any action. 

We also need to have Infra bless any 3rd party plugins we use. I filed https://issues.apache.org/jira/browse/INFRA-26051 for this.

Reply:
Infra has signed off on this action 👍 

Do we specifically need code from the lastest code this action? I assume that we probably don't and could use a release tag instead like:

```
uses: aquasecurity/trivy-action@0.24.0
```

Reply:
The best practice says to pin to a specific SHA https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions#using-third-party-actions

Reply:
I modified the PR to point to commit SHA of tag 0.24.0

Reply:
Done.